argus/version

```diff
@@ -1 +1 @@
-0.13.1
+0.14.1
```

Comment:
0.14.2?

Reply:
yes it would, didn't got yet to update that one, had issue in other fronts.

Reply:
updated it